### DISCUSSION TITLE
Posting Image Blob alongside post

### DISCUSSION THREAD

**Discussion by ixnoahlive** [bluesky-social/atproto#3245] [How to do it?]
I've got a url to an image and converted it to a image blob, but how do I actually attach it alongside the post? There's a bunch of conveluded things and all documentation uses Python instead of TypeScript so I have no clue what to do to get my Blob into a functioning image attached to my post.

**Comment by bnewbold** [comment]
There is TypeScript code here in the "posts" tutorial:

https://docs.bsky.app/docs/tutorials/creating-a-post#images-embeds

We have a new specification document which might be helpful:

https://atproto.com/specs/blob
